src/components/Patient/PatientRegistration.tsx

```diff
@@ -471,7 +471,7 @@ export default function PatientRegistration(
                   ["dob", t("date_of_birth")],
                   ["age", t("age")],
                 ].map(([key, label]) => (
-                  <TabsTrigger value={key}>{label}</TabsTrigger>
+                  <TabsTrigger value={key as string}>{label}</TabsTrigger>
```

Comment:
pass `as const` to the array before calling the .map. So that compiler can determine the type of keys without we needing to tell it.

```suggestion
                  <TabsTrigger value={key}>{label}</TabsTrigger>
```